clients/js/packages/client/src/client.ts

```diff
@@ -55,6 +55,7 @@ const defaultPorts: PodPorts = {
   chains: {
     defaultPorts: {
       rpc: 26657,
+      grpc:30657,
```

Comment:
@Kayanski thanks for opening this PR up. Sorry to have missed this.
I think the default grpc port for the chain is `9090`, could we change it to that? Then we are good to go.

```suggestion
      grpc:9090,
```